src/DynamoCore/Graph/Nodes/NodeInputData.cs

```diff
@@ -27,7 +57,10 @@ public enum NodeInputTypes
         [EnumMember(Value = "hostSelection")]
         hostSelection,
         [EnumMember(Value = "dropdownSelection")]
-        dropdownSelection
+        dropdownSelection,
+        //TODO could also just set this to 0 since this is a new enum, and get rid of default deserialization functionality.
+        [EnumMember(Value = "unknown")]
```

Comment:
the `[EnumMember(Value = "unknown")]` attribute is used by the JSON serializer ? this will be the name stored on disk ?